src/api/app/views/webui/staging/workflows/_overall_state.html.haml

```diff
@@ -1,8 +1,9 @@
-%span.badge{ class: "state-#{staging_project.overall_state}" }
-  %span.badge.badge-light
-    = link_to(staging_project.name, staging_workflow_staging_project_path(staging_workflow, staging_project.name))
-  %span
-    %br
-    = staging_project.overall_state
-    %br
+%div{ class: "overall-state staging-project state-#{staging_project.overall_state}" }
+  .letter
+    = link_to(staging_project.staging_nick, staging_workflow_staging_project_path(staging_workflow, staging_project.name))
+  .state
+    - state = staging_project.overall_state
+    - state = 'broken' if state == :unacceptable
```

Comment:
Possibly we can rename the whole state - 'empty' and 'acceptable' are the only states that are interesting to the client.